To: vim_dev@googlegroups.com
Subject: Patch 7.3.104
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Problem:    Conceal: using Tab for cchar causes problems. (ZyX)
Solution:   Do not accept a control character for cchar.
Files:      src/syntax.c


*** ../vim-7.3.103/src/syntax.c	2010-09-29 18:32:47.000000000 +0200
--- src/syntax.c	2011-01-22 00:50:20.000000000 +0100
***************
*** 4537,4542 ****
--- 4537,4549 ----
  		;
  #endif
  	    }
+ #ifdef FEAT_CONCEAL
+ 	    if (!vim_isprintc_strict(*conceal_char))
+ 	    {
+ 		EMSG(_("E844: invalid cchar value"));
+ 		return NULL;
+ 	    }
+ #endif
  	    arg = skipwhite(arg + 7);
  	}
  	else
*** ../vim-7.3.103/src/version.c	2011-01-22 00:11:42.000000000 +0100
--- src/version.c	2011-01-22 00:54:56.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     104,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
132. You come back and check this list every half-hour.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
